controllers/constant/odlm.go

```diff
@@ -442,18 +442,58 @@ metadata:
 spec:
   services:
   - name: ibm-platformui-operator-v4.0
+    resources:
+      - apiVersion: apps/v1
+        force: true
+        kind: Deployment
+        labels:
+          operator.ibm.com/opreq-control: 'true'
+        name: meta-api-deploy
+        namespace: {{ .OperatorNs }}
     spec:
       operandBindInfo: {}
   - name: ibm-platformui-operator-v4.1
+    resources:
+      - apiVersion: apps/v1
+        force: true
+        kind: Deployment
+        labels:
+          operator.ibm.com/opreq-control: 'true'
+        name: meta-api-deploy
+        namespace: {{ .OperatorNs }}
     spec:
       operandBindInfo: {}
   - name: ibm-platformui-operator-v4.2
+    resources:
+      - apiVersion: apps/v1
+        force: true
+        kind: Deployment
+        labels:
+          operator.ibm.com/opreq-control: 'true'
+        name: meta-api-deploy
+        namespace: {{ .OperatorNs }}
     spec:
       operandBindInfo: {}
   - name: ibm-platformui-operator-v4.3
+    resources:
+      - apiVersion: apps/v1
+        force: true
+        kind: Deployment
+        labels:
+          operator.ibm.com/opreq-control: 'true'
+        name: meta-api-deploy
+        namespace: {{ .OperatorNs }}
     spec:
       operandBindInfo: {}
   - name: ibm-platformui-operator-v4.4
+    resources:
+      - apiVersion: apps/v1
+        force: true
+        kind: Deployment
+        labels:
+          operator.ibm.com/opreq-control: 'true'
+        name: meta-api-deploy
+        namespace: {{ .OperatorNs }}
```

Comment:
```suggestion
        namespace: "{{ .CPFSNs }}"
```
Hi, would you mind replacing all  `{{ .OperatorNs }}` with `"{{ .CPFSNs }}"` for all `ibm-platformui-operator-v4.x`
according to https://github.com/IBM/ibm-common-service-operator/pull/2279/files